### PR TITLE
Update M&T Bank formatting

### PR DESCRIPTION
**Pull request by mmomjian**
"and" -> "&" as per their logo and website branding

### CODE REVIEW

File: _data/banking.yml [hunk 1]
```diff
@@ -908,7 +908,7 @@ websites:
       - hardware
     doc: https://www.lansforsakringar.se/privat/att-vara-kund/digitala-tjanster-telefonbank/internetbanken/
 
-  - name: MandT Bank
+  - name: M&T Bank
```
Comment (phallobst) [inline _data/banking.yml]: I think it's better to use this encoding for HTML entities:
```suggestion
  - name: M&amp;T Bank
```

**Review by phallobst — CHANGES_REQUESTED** [review]

**Comment by mmomjian** [comment]
Makes sense, I had just grabbed it from the closest entry, which is "-name: Swedbank & Sparbankerna"

**Review by phallobst — APPROVED** [review]

**Review by ApCoder123 — CHANGES_REQUESTED** [review]
Please use this PNG with a fully transparent background.

mtbank

**Comment by mmomjian** [comment]
@ApCoder123 done!

**Review by ApCoder123 — APPROVED** [review]

**Pull request merged**